index.html

```diff
@@ -1,91 +1,62 @@
-                        
-                            <select id="from-currency-dropdown">
-							<option value="">Select One …</option>
-							</select>
-							<label>Currency Code:</label>
+<body onload="populateCurrencyDropdown()">
+    <h2>Insert a New Currency Rate</h2>
+    <form id="insertCurrencyForm">
+        <label for="base">Base Currency:</label><br>
+        <input type="text" id="base" name="base" value="EUR" disabled><br>
+        <label for="targetCurrency">Target Currency:</label><br>
+        <input type="text" id="targetCurrency" name="targetCurrency"><br>
+        <label for="rate">Rate:</label><br>
+        <input type="number" step="0.00001" id="rate" name="rate"><br><br>
+        <input type="button" value="Insert Rate" onclick="insertCurrencyRate()">
+    </form>
+    <pre id="newRateOutput"></pre>
+
+    <hr/>
+
+    <h2>Convert Currency</h2>
+    <form id="convertCurrencyForm">
+        <label for="fromCurrency">From:</label><br>
+        <select id="fromCurrency" name="fromCurrency">
+            <option value="">Select Currency</option>
+        </select>
+        <label> 1 </label>
 									<label id="selected-from-currency-label"></label>
-                        </div>
-                    </div>
-     
-                    <div class="col-sm-6">
-                        <div class="input-group mb-3">
-                            <div class="input-group-prepend">
-                                <span class="input-group-text">To</span>
-                            </div>
-                            <select id="to-currency-dropdown">
-							<option value="">Select One …</option>
-							</select>
-                        </div>
-                    </div>
-                </div>
-				
-				<br/>
-<br/>
-     
-                <div class="text-center">
-                    
-     
-                    <!-- convert button -->
-                    <button class="btn btn-primary convert m-2" type="submit">
-                        Convert
-                    </button>
-                    <!-- reset button -->
-                    <button class="btn btn-primary m-2" onclick="clearVal()">
-                        Reset
-                    </button>
-                </div>
-     
-            </div>
-            <div calss="text-center2">
-                <p>Exchange rate on: <label id="datetime-label"/></p>
-                
-            </div>
+        <br>
+        <label for="toCurrency">To:</label><br>
+        <select id="toCurrency" name="toCurrency">
+            <option value="">Select Currency</option>
+        </select><label> Equivalent to </label>
+                                    <label id="selected-to-currency-label"></label>
+        <br>
+        <label for="amount">Amount:</label><br>
+        <input type="number" step="0.00001" id="amount" name="amount"><br><br>
+        <input type="button" value="Convert" onclick="convertCurrency()"><label id="converted-amount-label"> </label>
```

Comment:
There is a `<button>` element in HTML. `<input type="button">` is a bad practice.
Plus, the same comment about `addEventListener`